src/site/content/en/learn/pwa/complexity/index.md

```diff
@@ -1,11 +1,126 @@
 ---
 title: Complexity management
-description: >
-  Keep a web app simple is more complicated than saying it. In this chapter, you will understand how to leverage web APIs to work with threading and state management in the PWA world.
-authors:
+Authors:
   - firt
-date: 2021-11-03
-placeholder: true
+description: >
+  Keeping a web app simple can be surprisingly complicated. In this module, you will learn how web APIs work with threading and how you can use this for common PWA patterns such as state management.
+date: 2022-04-15
 ---
 
-Coming soon!
+## Simplicity and complexity
+
+In his talk [Simple Made Easy](https://www.infoq.com/presentations/Simple-Made-Easy/), Rich Hickey discusses the qualities of simple versus complex things. He describes simple things as focusing on:
+
+* One role, one task, one concept, or one dimension.
+
+But emphasizes that simplicity isn't about:
+
+* Having one instance or performing one operation.
```

Comment:
```suggestion
> "Having one instance or performing one operation."
```

Reply:
these two have ended up as a single item list each, they seem to actually be quotes so maybe what I've done works, I don't think they should be list items though.